pic32/variants/Max32/Board_Defs.h

```diff
@@ -300,6 +300,10 @@ extern const uint32_t   digital_pin_to_cn_PGM[];
 #define _SER0_IPL_ISR	IPL2SOFT
 #define	_SER0_IPL		2
 #define	_SER0_SPL		0
+#define _SER0_TX_PIN    1
+#define _SER0_RX_PIN    0
+#define _SER0_RTS_PIN   18
```

Comment:
One thing to be aware of  here is that the `_SER0` flow control pins are not connected to the FTDI chip, so cannot be used at all in USB serial no matter what the software supports